packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/main/nodejs/integrationsteps.md

```diff
@@ -24,11 +24,11 @@ const transaction = await authClient.idx.authenticate({ username, password });
 
 ### Your app handles an authentication success response
 
-`authenticate()` returns a `transaction` object with a `status` property indicating the current state of the sign-in flow. Handle the returned `IdxStatus` value accordingly:
+`authenticate()` returns a `transaction` object with a `status` property, indicating the current state of the sign-in flow. Handle the returned `IdxStatus` value accordingly:
 
 #### Success status
 
-When the user correctly supplies their password, `IdxStatus` equals `IdxStatus.SUCCESS`. Call `tokenManager.setTokens()` to save the tokens retrieved from the response for future requests, and then redirect the user back to the home page. The user is now signed in.
+Once the user correctly enters their `password`, `IdxStatus` equals `IdxStatus.SUCCESS`. Call `tokenManager.setTokens()` to save the tokens retrieved from the response for future requests, and then redirect the user back to the home page. The user is now signed in.
```

Comment:
Once can suggest simultaneous action. We need to eliminate this type of ambiguity for translations.